schema/scheme.json

```diff
@@ -43,9 +43,15 @@
             "type": "string"
         },
         "nist round": {
-            "type": "integer",
-            "minimum": 0,
-            "$comment": "0 for schemes that were not submitted to the NIST PQC Standardization Process."
+            "type": "string",
+            "enum": [
+                "none",
+                "1",
+                "2",
+                "3a",
+                "3f"
+            ],
+            "$comment": "none for schemes that were not submitted to the NIST PQC Standardization Process."
```

Comment:
Shouldn't we here include information about `3a` and `3f`?

Reply:
Agreed, done